buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -0,0 +1,46 @@
+object SpringBoot {
```

Comment:
It's a little bit too kotlinerized. What's the advantage of such implementation? Is this a common practice somewhere?

Reply:
A trend, I'd say. Advatages I can see:
- consistent dependencies across modules
- code completion (!)
- improved `build.gradle` readability

![image](https://user-images.githubusercontent.com/11961746/76398254-da6ea400-637c-11ea-81da-dd4229dc26d5.png)

Reply:
![Peek 2020-03-04 13-49](https://user-images.githubusercontent.com/11961746/76398357-0853e880-637d-11ea-8505-66e28de3d557.gif)

Reply:
Code completion is of no value after the input has been made though, and given how infrequently anyone is likely to add dependencies, well.. 

It is also a tough sell to claim that readability is improved when 50% of the information of interest (the version) is no longer readable in-file. 

Remaining point is consistency in dependencies across modules, and that one I agree on.

Reply:
It's already a common practice to have your versions stored in `settings.gradle` so you have a separate file with versions, so this is just improved version of that.

Reply:
This is in response to your video, where the version is clearly seen before being overwritten. I am sure there are benefits in huge multi-module projects, but for anything else I'd prefer to keep things as simple as possible. Your choice though.